keepassxc-browser/common/sites.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+const PREDEFINED_SITELIST = [
+    'https://accounts.google.com/*',
+    'https://www.paypal.com/*',
+    'https://*.live.com/*',
+    'https://www.amazon.com/*',
+    'https://www.amazon.de/*',
+    'https://www.amazon.co.uk/*',
+    'https://signin.aws.amazon.com/*',
+    'https://www.upwork.com/ab/*',
+    'https://home.personalcapital.com/*',
+    'https://auth.services.adobe.com/*',
+    'https://idmsa.apple.com/*',
+    'https://*.soundcloud.com/*',
+    'https://icloud.com/*',
+    'https://signin.ebay.de/*',
+    'https://signin.ebay.com/*',
+    'https://signin.ebay.com.au/*',
+    'https://signin.ebay.com.cn/*',
+    'https://signin.ebay.com.hk/*',
+    'https://signin.ebay.com.my/*',
+    'https://signin.ebay.com.sg/*',
+    'https://signin.ebay.it/*',
+    'https://signin.ebay.co.uk/*',
+    'https://signin.ebay.ca/*',
+    'https://signin.ebay.at/*',
+    'https://signin.ebay.be/*',
+    'https://signin.ebay.fr/*',
+    'https://signin.ebay.ie/*',
+    'https://signin.ebay.nl/*',
+    'https://signin.ebay.es/*',
+    'https://signin.ebay.ch/*',
+    'https://signin.ebay.in/*',
+    'https://signin.ebay.ph/*',
+    'https://login.yahoo.com/*'
+];
+
+const kpxcSites = {};
+kpxcSites.googlePasswordFormUrl = 'https://accounts.google.com/signin/v2/challenge/password';
+kpxcSites.googleUrl = 'https://accounts.google.com';
+kpxcSites.savedForm = undefined;
+
+// Handles a few exceptions for certain sites where password form is inside a div
+// or another element that is not detected directly. Triggered by MutationObserver.
+// Returns true if an Element has a matching classList.
+kpxcSites.exceptionFound = function(classList) {
```

Comment:
Can we use this with AWS?

Reply:
Yes. Although I haven't been able to reproduce the issue some are having.

Reply:
I'll test with this branch to see if it works

Reply:
It works for the username field, but password refuses to fill

Reply:
And for me, everything works OK.

Reply:
Bigger fish to fry atm